modules/nlu/src/backend/engine.ts

```diff
@@ -55,9 +61,26 @@ export default class ScopedEngine {
     if (isNaN(this.confidenceTreshold) || this.confidenceTreshold < 0 || this.confidenceTreshold > 1) {
       this.confidenceTreshold = 0.7
     }
+
+    if (!isNaN(this._autoSyncInterval) && this._autoSyncInterval >= 5000) {
```

Comment:
should add `&& !!_this.autoSyncInterval` just to make sure it's not null.  `isNaN(null) == false`